lua/lvim/utils/modules.lua

```diff
@@ -95,4 +95,59 @@ M.reload = function(mod)
   return old
 end
 
+-- code from <https://github.com/tjdevries/lazy-require.nvim/blob/bb626818ebc175b8c595846925fd96902b1ce02b/lua/lazy-require.lua#L25>
+function M.require_on_index(require_path)
+  return setmetatable({}, {
+    __index = function(_, key)
+      return require(require_path)[key]
+    end,
+
+    __newindex = function(_, key, value)
+      require(require_path)[key] = value
+    end,
+  })
+end
+
+-- code from <https://github.com/tjdevries/lazy-require.nvim/blob/bb626818ebc175b8c595846925fd96902b1ce02b/lua/lazy-require.lua#L25>
+function M.require_on_exported_call(require_path)
+  return setmetatable({}, {
+    __index = function(_, k)
+      return function(...)
+        return require(require_path)[k](...)
+      end
+    end,
+  })
+end
+
+function M.lazy_set(require_path, fn_name)
```

Comment:
did you write this manually? isn't this one missing `__call`?

Reply:
yes and yes

Reply:
> although it doesn't need __call, it makes more sense to use `require_on_exported_call` in that case

I think you forgot to add that one :eyes:

[wouldn't it be the one to use with telescope?](https://github.com/tjdevries/lazy-require.nvim/blob/bb626818ebc175b8c595846925fd96902b1ce02b/lua/lazy-require.lua#L56-L61)

Reply:
I did change telescope to use `require_on_exported_call`

`lazy_set` is only used for dashboard config table so that it's loaded with the plugin or if the user wants to change something in the table